### DISCUSSION TITLE
Best practice for zstash update... remove index.db first?

### DISCUSSION THREAD

**Discussion by jjbenedict** [E3SM-Project/zstash#350] [General]
I'm using zstash on Perlmutter to archive an E3SM simulation to HPSS.  After the simulation completed 37 years, I successfully ran the short-term archiver and then `zstash create` (and `zstash check`).  I then extended the simulation to 44 years and again ran the short-term archiver.  I'd now like to run `zstash update` to update the HPSS archive.  My case directory contains the usual subdirectories (archive, build, run), but it also contains `zstash` which includes `index.db` from the earlier `zstash create` step.  Should I remove this existing `index.db` file before running `zstash update`?

**Comment by forsyth2** [comment]
> Should I remove this existing `index.db` file before running `zstash update`?

No, that is fine. Actually, that is what will tell `zstash` what is already archived from the `create` step. One thing to note is that `zstash update` should be run from inside the directory you're archiving, rather than passing that directory in as the final argument, as is done with `zstash create`.
